main.go

```diff
@@ -69,58 +75,57 @@ func main() {
 	time.Sleep(300 * time.Hour)
```

Comment:
configurable? In case we need to adjust intervals in the future

Reply:
Good catch will add a forever loop